Libraries/Lists/VirtualizedList.d.ts

```diff
@@ -154,6 +166,11 @@ export interface VirtualizedListWithoutRenderItemProps<ItemT>
     | null
     | undefined;
 
+  /**
+   * Styling for internal View for ListHeaderComponent
+   */
+  ListHeaderComponentStyle?: StyleProp<ViewStyle> | undefined;
```

Comment:
It looks like this is separately defined in [`FlatList.d.ts`](https://github.com/facebook/react-native/blob/6410a621cedc122352ed7453ea920464331fda50/Libraries/Lists/FlatList.d.ts) and [`SectionList.d.ts`](https://github.com/facebook/react-native/blob/8cdc9e7f04e2dd3026d15dcd5765952bfb2c5c08/Libraries/Lists/SectionList.d.ts). These copies can now be removed, since they inherit from `VirtualizedListProps`.

Reply:
Done. also remove ListFooterComponent and ListHeaderComponent from them.